packages/trader/src/sass/app/modules/reports.scss

```diff
@@ -96,6 +96,12 @@ $side-padding: 1.2em;
 .statement {
     .table__head {
         font-weight: bold;
+        white-space: normal;
```

Comment:
should this apply for both `desktop` and `mobile` viewports?

Reply:
desktop only should be fine, let me change 🙏